FreeRTOS/FreeRTOS
# MQTT mutual auth demo.

MQTT mutual auth demo.

Description
-----------
MQTT mutual auth demo.

Demo for showing use of the managed MQTT API over a mutually authenticated connection.

The Example shown below uses this API to create MQTT messages and
send them over the connection established using FreeRTOS sockets.
The example is single threaded and uses statically allocated memory;
it uses QoS1.

A mutually authenticated TLS connection is used to connect to the AWS IoT
MQTT message broker in this example. Define democonfigAWS_ROOT_CA_PEM,
democonfigCLIENT_CERTIFICATE_PEM, and democonfigCLIENT_PRIVATE_KEY_PEM in
demo_config.h to achieve mutual authentication.

Test Steps
-----------
1. Ran the demo.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
